### ISSUE TITLE
The performance in Table

### ISSUE THREAD

**Issue by buaaswf** [huanghoujing/person-reid-triplet-loss-baseline#8]
Hi @huanghoujing ,
I am training the Mkaert1501 dataset only, but the performance is worse than your table's.
Are the results in the table training on all the datasets combined, or I have done something wrong.
I do as the following steps:
1
python script/dataset/transform_market1501.py \
--zip_file ~/Dataset/market1501/Market-1501-v15.09.15.zip \
--save_dir ~/Dataset/market1501
2
python script/experiment/train.py \
-d '(0,)' \
--only_test false \
--dataset DATASET_NAME \
--last_conv_stride STRIDE \
--normalize_feature false \
--trainset_part TRAINVAL_OR_TRAIN \
--exp_dir EXPERIMENT_DIRECTORY \
--steps_per_log 10 \
--epochs_per_val 5

**Comment by buaaswf** [comment]
`=========> Test on dataset: market1501 <=========

Extracting feature...
1000/1000 batches done, +1.59s, total 84.73s
Done, 85.02s
Computing distance...
Done, 1.29s
Computing scores...
User Warning: Version 0.18.1 is required for package scikit-learn, your current version is 0.19.1. As a result, the mAP score may not be totally correct. You can try `pip uninstall scikit-learn` and then `pip install scikit-learn==0.18.1`
Done, 14.85s
Single Query:                 [mAP: 58.49%], [cmc1: 77.58%], [cmc5: 90.74%], [cmc10: 94.18%]
Multi Query, Computing distance...
Done, 1.71s
Multi Query, Computing scores...
User Warning: Version 0.18.1 is required for package scikit-learn, your current version is 0.19.1. As a result, the mAP score may not be totally correct. You can try `pip uninstall scikit-learn` and then `pip install scikit-learn==0.18.1`
Done, 14.56s
Multi Query:                  [mAP: 67.08%], [cmc1: 84.12%], [cmc5: 94.45%], [cmc10: 96.59%]
Re-ranking distance...
Done, 95.28s
Computing scores for re-ranked distance...
User Warning: Version 0.18.1 is required for package scikit-learn, your current version is 0.19.1. As a result, the mAP score may not be totally correct. You can try `pip uninstall scikit-learn` and then `pip install scikit-learn==0.18.1`
Done, 15.77s
Re-ranked Single Query:       [mAP: 74.54%], [cmc1: 81.32%], [cmc5: 89.49%], [cmc10: 91.89%]
Multi Query, Re-ranking distance...
Done, 90.14s
Multi Query, Computing scores for re-ranked distance...
User Warning: Version 0.18.1 is required for package scikit-learn, your current version is 0.19.1. As a result, the mAP score may not be totally correct. You can try `pip uninstall scikit-learn` and then `pip install scikit-learn==0.18.1`
Done, 15.77s
Re-ranked Multi Query:        [mAP: 81.72%], [cmc1: 87.44%], [cmc5: 93.50%], [cmc10: 95.28%]
`

**Comment by huanghoujing** [comment]
Thanks for attention!
I train and test on single dataset, without combining datasets. The scores in the table takes 300 epochs to train. You may set the argument `exp_decay_at_epoch` to `150`, `total_epochs` to `300` and re-try.
